src/main/g8/src/main/scala/$package__packaged$/HelloWorldRoutes.scala

```diff
@@ -6,10 +6,14 @@ import org.http4s.HttpRoutes
 import org.http4s.circe._
 import org.http4s.dsl.Http4sDsl
 
-class HelloWorldRoutes[F[_]: Sync] extends Http4sDsl[F] {
-  val routes: HttpRoutes[F] =
+object HelloWorldRoutes {
+
+  def routes[F[_]: Sync]: HttpRoutes[F] = {
+    val dsl = new Http4sDsl[F]{}
+    import dsl._
```

Comment:
I mostly like the structure of this.  This is the part I don't.  Maybe we need a poll or something.

Reply:
I stand by this, I prefer it as I don't need to need to instantiate anything else except the dsl here. Otherwise ghe type parameter leaks up an inheritance hierarchy. With this it's a single function. I could instantiate and pass if you prefer.

Reply:
For client syntax, I now do `: MyThing[F] = new MyThing[F] with Http4sClientDsl[F] {`
. It's a shame that we can't replicate entirely it on the server, I do think it strikes a nice balance

Reply:
The client builders are parameterized classes, where as everything here is objects with parameterized methods. Is this difference justifiable or is one more right?